### ISSUE TITLE
scmdata compatibility

### ISSUE THREAD

**Issue by mikapfl** [pik-primap/primap2#149]
It would be great to import data from scmdata into primap2 and vice versa.

**Comment by JGuetschow** [comment]
Yes it would. It's not super easy though as scmdata has this terrible (in my opinion) format where values for different dimensions are concatenated into a string. And we have to think about also exchanging terminologies / vocabularies.
